src/watertap_contrib/reflo/unit_models/zero_order/battery.py

```diff
@@ -0,0 +1,283 @@
+#################################################################################
```

Comment:
Didn't realize this came with a battery ZO model (which I fully support). However, we _should_ have at least a small test file associated with it.

Reply:
Nevermind, I see this is under ``code_demos/unit_models`` rather than our main ``unit_models`` folder. So I am ok with no test file.

Reply:
So this was under code_demos/unit_models, but one of Adam's comments was to put it in the unit_models folder so  I moved it into the unit_models/zero_order folder. So I'll add a quick test